src/main/java/com/kuliginstepan/dadata/client/util/DadataWebClientBuilder.java

```diff
@@ -0,0 +1,26 @@
+package com.kuliginstepan.dadata.client.util;
+
+import io.netty.handler.timeout.ReadTimeoutHandler;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.client.reactive.ReactorClientHttpConnector;
+import org.springframework.web.reactive.function.client.WebClient;
+import reactor.netty.http.client.HttpClient;
+import reactor.netty.tcp.TcpClient;
+
+import java.time.Duration;
+import java.util.concurrent.TimeUnit;
+
+public class DadataWebClientBuilder {
+
+    public static WebClient buildDadataWebClient(String url, String token, Duration timeout) {
```

Comment:
What if we change signature of this method to take 1 parameter of type `DadataClientProperties`?

Reply:
I thought about it, but if this utility method will be used by someone else for some reason, it will be not quite comfortable to him to create instance of `DadataClientProperties`, then fill it and use in this method.

Maybe we can just add method receiving `DadataClientProperties`which will call existing one?